bench-cargo-miri/serde2/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "cargo-miri-test"
+version = "0.1.0"
+authors = ["Oliver Schneider <git-spam-no-reply9815368754983@oli-obk.de>"]
+edition = "2018"
+
+[dependencies]
+serde = { version = "*", features = ["derive"] }
+serde_json = "*"
```

Comment:
We have lockfiles check in, but I feel like `*` is not what we want here

Reply:
:shrug: this is a literal copy of the serde1 test. But yeah I'll put in something less silly.